assets/sass/components/settings/_googlesitekit-settings-notice.scss

```diff
@@ -21,109 +21,100 @@
-				background-size: 20px 19px;
-				border-color: $c-icon-notice-info;
-				border-radius: 0;
-				border-width: 0;
-				height: 19px;
-				width: 20px;
+			@media (min-width: $bp-desktop) {
+				padding-top: $grid-gap-desktop;
 			}
+		}
 
-			> div + div {
-				margin-top: 32px;
-			}
+		.googlesitekit-settings-notice__icon {
+			margin-right: $grid-gap-desktop;
 		}
 
-		.mdc-select:not(.mdc-select--disabled) {
+		.googlesitekit-settings-notice__text {
+			flex-grow: 1;
 
-			.mdc-floating-label {
-				color: $c-text-notice-info-selected-text;
-			}
+			.mdc-select:not(.mdc-select--disabled) {
 
-			.mdc-select__selected-text {
-				color: $c-text-notice-info-selected-text;
-			}
+				.mdc-floating-label {
+					color: $c-text-notice-info-selected-text;
+				}
 
-			.mdc-select__dropdown-icon {
-				background-image: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%231a73e8%22%20fill-rule%3D%22evenodd%22%20opacity%3D%221%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
-			}
+				.mdc-select__selected-text {
+					color: $c-text-notice-info-selected-text;
+				}
 
-			.mdc-notched-outline__leading,
-			.mdc-notched-outline__notch,
-			.mdc-notched-outline__trailing {
-				border-color: $c-border-brand;
-			}
+				.mdc-select__dropdown-icon {
+					background-image: url("data:image/svg+xml,%3Csvg%20width%3D%2210px%22%20height%3D%225px%22%20viewBox%3D%227%2010%2010%205%22%20version%3D%221.1%22%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%3E%0A%20%20%20%20%3Cpolygon%20id%3D%22Shape%22%20stroke%3D%22none%22%20fill%3D%22%231a73e8%22%20fill-rule%3D%22evenodd%22%20opacity%3D%221%22%20points%3D%227%2010%2012%2015%2017%2010%22%3E%3C%2Fpolygon%3E%0A%3C%2Fsvg%3E");
+				}
 
-			&:not(.mdc-select--focused) {
+				.mdc-notched-outline__leading,
+				.mdc-notched-outline__notch,
+				.mdc-notched-outline__trailing {
+					border-color: $c-border-brand;
+				}
+
+				&:not(.mdc-select--focused) {
 
-				.mdc-select__selected-text:hover ~ .mdc-notched-outline {
+					.mdc-select__selected-text:hover ~ .mdc-notched-outline {
 
-					.mdc-notched-outline__leading,
-					.mdc-notched-outline__notch,
-					.mdc-notched-outline__trailing {
-						border-color: $c-border-notice-info-hover;
+						.mdc-notched-outline__leading,
+						.mdc-notched-outline__notch,
+						.mdc-notched-outline__trailing {
+							border-color: $c-border-notice-info-hover;
+						}
 					}
 				}
 			}
 		}
+
+		.googlesitekit-settings-notice__learn-more {
+			flex-shrink: 0;
+			margin-left: $grid-gap-desktop;
+		}
```

Comment:
@danielgent we need to make sure that the "learn more" link is displayed correctly on smaller screens. Please, test settings notice styles for responsiveness.

Also, please, pay attention to the GA4PropertyNotice component, it should display the LearnMore link in the right bottom corner. If you look at the storybook stories, it shows the LearnMore link in the center: https://google.github.io/site-kit-wp/storybook/pull/3404/?path=/story/modules-analytics-setup-setupformga4transitional--without-existing-tag

Reply:
Ah @eugene-manuilov , this is what those wrapping `div`s were for then that I removed :joy: 

I'll have to have another look

Reply:
No it wasn't that, `children` needed a wrapping div. Now added
![fixed](https://user-images.githubusercontent.com/8496063/121178427-c30fb680-c85e-11eb-9df3-935643b8278f.png)

Reply:
@eugene-manuilov  regarding mobile

Currently, this is what it looks like on `develop` at about 320px (not counting the whitespace of course!)

![mobile](https://user-images.githubusercontent.com/8496063/121179034-7d9fb900-c85f-11eb-95ed-873eca32a07a.png)

As we have no mobile designs I'll spend some time now making everything collapse into single columns if needed :+1: